src/subcommands/diff.rs

```diff
@@ -39,7 +41,7 @@ pub fn diff(
     };
 
     let diff_process = process::Command::new(diff_path)
-        .args(&diff_cmd[1..])
+        .args(diff_cmd)
         .args([minus_file, plus_file])
         .stdout(process::Stdio::piped())
         .spawn();
```

Comment:
`git diff --no-index --nonsense` will exit with 129 and first print `error: unknown option 'nonsense'` and then the _full_ `--help` output. Maybe catch such cases (`diff` will just exit with 2) and print the actual (quoted, in case `split_whitespace()` was the culprit) command `delta` called.

Reply:
I haven't done this yet. This could also be where we catch people using git <2.43 with process substitution and fallback to diff?

Reply:
Done, see https://github.com/dandavison/delta/pull/1697/files#r1703205451